### PR TITLE
CE-Support&Compatibility-updates-3.16

### PR DESCRIPTION
**Pull request by bmckercher123**
### Updates for 3.16 release 

- For 3.16/next

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx [hunk 1]
```diff
@@ -138,6 +144,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | OpenShift versions for Kubernetes | {{prodname}} support                 |
 | -------------------- | --------------------------------- | ------------------------------------ |
+| 3.16                 | 4.9 - 4.11                        | {{prodname}} CNI with network policy |
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx]:
```suggestion
| 3.16                 | 4.10 - 4.12                        | {{prodname}} CNI with network policy |
```

File: calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx [hunk 2]
```diff
@@ -89,15 +92,16 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 ## kOps on AWS
 
-| {{prodname}} version | kOps and Kubernetes versions | {{prodname}} support                                                                   |
-| -------------------- | ---------------------------- | -------------------------------------------------------------------------------------- |
+| {{prodname}} version | kOps and Kubernetes versions | {{prodname}} support                                         |
+| -------------------- | ---------------------------- | ------------------------------------------------------------ |
+| 3.16                 | 1.23 - 1.25                  | - {{prodname}} CNI with network policy<br />- AWS CNI with {{prodname}} network policy |
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx]:
```suggestion
| 3.16                 | 1.24 - 1.25                  | - {{prodname}} CNI with network policy<br />- AWS CNI with {{prodname}} network policy |
```

File: calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx [hunk 3]
```diff
@@ -106,6 +110,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | Kubernetes/kubeadm versions | {{prodname}} support                 |
 | -------------------- | --------------------------- | ------------------------------------ |
+| 3.16                 | 1.23 - 1.25                 | {{prodname}} CNI with network policy |
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx]:
```suggestion
| 3.16                 | 1.24 - 1.26                 | {{prodname}} CNI with network policy |
```

File: calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx [hunk 4]
```diff
@@ -154,6 +161,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | RKE version | {{prodname}} support                 | Kubernetes versions |
 | -------------------- | ----------- | ------------------------------------ | ------------------- |
+| 3.16                 | 1.3         | {{prodname}} CNI with network policy | 1.23                |
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx]:
```suggestion
| 3.16                 | 1.4         | {{prodname}} CNI with network policy | 1.23                |
```

File: calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx [hunk 5]
```diff
@@ -168,15 +176,17 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 ## RKE2
 
-| {{prodname}} version | {{prodname}} support                 | Kubernetes versions |
-| -------------------- | ------------------------------------ | ------------------- |
-| 3.15                 | {{prodname}} CNI with network policy | 1.24                |
-| 3.14                 | {{prodname}} CNI with network policy | 1.22                |
+| {{prodname}} version | RKE2 version    | {{prodname}} support                 | Kubernetes versions |
+| -------------------- | --------------- | ------------------------------------ | ------------------- |
+| 3.16                 | 2.6.5 and later | {{prodname}} CNI with network policy | 1.24                |
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx]:
```suggestion
| 3.16                  | {{prodname}} CNI with network policy | 1.24 - 1.25                |
```

File: calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx [hunk 6]
```diff
@@ -168,15 +176,17 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 ## RKE2
 
-| {{prodname}} version | {{prodname}} support                 | Kubernetes versions |
-| -------------------- | ------------------------------------ | ------------------- |
-| 3.15                 | {{prodname}} CNI with network policy | 1.24                |
-| 3.14                 | {{prodname}} CNI with network policy | 1.22                |
+| {{prodname}} version | RKE2 version    | {{prodname}} support                 | Kubernetes versions |
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx]:
```suggestion
| {{prodname}} version | {{prodname}} support                 | Kubernetes versions |
```

File: calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx [hunk 7]
```diff
@@ -168,15 +176,17 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
...
+| -------------------- | --------------- | ------------------------------------ | ------------------- |
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx]:
```suggestion
| -------------------- | ------------------------------------ | ------------------- |
```

File: calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx [hunk 8]
```diff
@@ -168,15 +176,17 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
...
+| 3.16                 | 2.6.5 and later | {{prodname}} CNI with network policy | 1.24                |
+| 3.15                 | RKE2-compatible cluster | {{prodname}} CNI with network policy | 1.24                |
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx]:
```suggestion
| 3.15                 | {{prodname}} CNI with network policy | 1.24                |
```

File: calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx [hunk 9]
```diff
@@ -168,15 +176,17 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
...
+| 3.14                 | RKE2-compatible cluster             | {{prodname}} CNI with network policy | 1.22                |
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx]:
```suggestion
| 3.14                 | {{prodname}} CNI with network policy | 1.22                |
```

File: calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx [hunk 10]
```diff
@@ -168,15 +176,17 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
...
 
 ## TKG
 
 | {{prodname}} version | TKG version | {{prodname}} support                 | Kubernetes versions    |
 | -------------------- | ----------- | ------------------------------------ | ---------------------- |
+| 3.16                 | 1.5         | {{prodname}} CNI with network policy | 1.22                   |
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/compatibility.mdx]:
```suggestion
| 3.16                 | 1.6         | {{prodname}} CNI with network policy | 1.23                   |
```

**Review by sujeet-kr — DISMISSED** [review]
LGTM 🚀

**Comment by ctauchen** [comment]
Needs squashing first. Otherwise LGTM

File: __tests__/redirects.test.js [hunk 11]
```diff
@@ -1,41 +1,62 @@
...
+const http = require('node:http');
 
 test("Test old site to new site redirects", async () => {
+  const log = s => process.stdout.write(`${s}`);
   const WIP = 'wip', DONE = 'done', ERROR = 'error';
   const urlMap = new Map();
   const isFullReport = process.env.FULL_REPORT ? process.env.FULL_REPORT === 'true' : false;
-  const ax = axios.create({ maxRedirects: 0 });
+  const ax = axios.create({
+    maxRedirects: 0,
```
Comment (ctauchen) [inline __tests__/redirects.test.js:16]:
```suggestion
    maxRedirects: 17,
```

**Pull request closed without merging**